flow/designs/asap7/mock-array-big/io.tcl

```diff
@@ -45,5 +45,5 @@ set assignments [list \
 ]
 
 foreach {direction names} $assignments {
-    set_io_pin_constraint -region $direction:* -pin_names $names
+    set_io_pin_constraint -order -region $direction:* -pin_names $names
```

Comment:
This is not correct, as the `-order` flag is only used with the `-group` flag. I think PPL will not error out, but the `-order` flag will do nothing here.

Reply:
@maliberty That's strange. It seems completely reasonable to want to specify the region(which edge) and that the pins should be ordered.

Reply:
We don't have support for it yet, but it can be implemented.

Reply:
I see. To order many pins on an edge unoptimized(fallback because of too many pins) is our main use-case.